doc/source/general/kospartmodule.rst

```diff
@@ -0,0 +1,108 @@
+.. _kospartmodule:
+
+KOS Processor PartModule Configuration Fields
+=============================================
+
```

Comment:
This new file currently generates the following Sphinx warning, which is normally important:

/home/madings/GitHub/KOS-1/doc/source/general/kospartmodule.rst:: WARNING: document isn't included in any toctree

It means there is no mention of this file in any table of contents and therefore it is orphaned other than for deep links that point into sections of it.  That would normally be a big deal.  But in this case I believe one of the other doc PR's that hasn't been merged yet provides a change that moves to a new system where every directory automatically gets scanned for its contents to form a toc tree.  I think this warning goes away after everything is put together.

But just in case I'm wrong, this should be looked at later after everything is merged together.